examples/filters.go

```diff
@@ -13,8 +13,8 @@ func main() {
 	// login to Array, get groupService instance
 	groupService, err := service.NewNimbleGroupService(
 		service.WithHost("1.1.1.1"),
-		service.WithTenantUser("xxx"),
-		service.WithPassword("xxx"))
+ 		service.WithUser("xxx"),
```

Comment:
Was an extra space character added at the beginning of the line?  Also on the next line and in some of the other example files.

Reply:
Whitespace can be difficult to see, but the green square in the diff looks like an added space.

Reply:
Oh isn't that because WithTenantUser vs WithUser?

Reply:
I don't think so.  Notice how the password line also shows a change.  Similar in initiator_group.go and restore.go.